### PR TITLE
fix: deprecate

### PR DESCRIPTION
**Pull request by tjdevries**
Fix: #1

### CODE REVIEW

**Review by Sparkenstein — APPROVED** [review]

**Comment by drodger** [comment]
I'm so glad someone took the time to fork this project before it's too late! It's always sad when the original developer stops caring...

**Comment by Herz3h** [comment]
saved

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# git-worktree.nvim
\ No newline at end of file
+# git-worktree.nvim
+
+NOTE: This project has been abandonded. Looking for maintainers!
```
Comment (frandsoh) [inline README.md:3]: Fix wording

```suggestion
NOTE: This project has been shamelessly abandonded and left to die by @ThePrimeagen . Looking for maintainers!
```

**Review by frandsoh — CHANGES_REQUESTED** [review]
Fix wording

**Review by frandsoh — APPROVED** [review]

**Comment by RichardMcSorley** [comment]
LGTM

Joey nice

**Review by eruizc-dev — APPROVED** [review]
- [x] Make by TJ
 - [x] Looks professional
 - [x] It's clear
 - [x] Looks professional
 - [x] Made by TJ
I approve

**Review by RichardMcSorley — APPROVED** [review]

**Comment by ThePrimeagen** [comment]
Telescopic "cash" Johnson, could you please resolve conflicts.

**Comment by RichardMcSorley** [comment]
poggers poggers poggers poggers poggers poggers

**Comment by ThePrimeagen** [comment]
ok jokes over babes

**Pull request closed without merging**